demisto_sdk/commands/content_graph/objects/base_content.py

```diff
@@ -316,6 +311,46 @@ def from_path(
     def match(_dict: dict, path: Path) -> bool:
         pass
 
+    @property
+    def related_content(self) -> dict:
+        if not self.related_content_dict:
+            self.related_content_dict = self.get_related_content()
+            if self.old_base_content_object:
+                git_util = GitUtil()
+                remote, branch = git_util.handle_prev_ver(
+                    self.old_base_content_object.git_sha  # type: ignore[arg-type]
+                )
+                for file in self.related_content_dict.values():
+                    for path in file["path"]:
+                        status = git_util._check_file_status(path, remote, branch)
+                        file["git_status"] = (
+                            None
+                            if (not status and not file["git_status"])
+                            else GitStatuses(status)
+                        )
+        return self.related_content_dict
+
+    def get_related_file(self, file_type) -> str:
```

Comment:
you would always want to read the related file as text (str)?

Reply:
I think I'll change it to `get_related_text_file`